modules/microshift-deploying-a-load-balancer.adoc

```diff
@@ -137,15 +157,17 @@ nginx   LoadBalancer   10.43.183.104   192.168.1.241   81:32434/TCP   2m
 
 .Verification
 
-* The following command forms five connections to the example `nginx` application using the external IP address of the `LoadBalancer` service configuration. The result of the command is a list of those server IP addresses. Verify that the load balancer sends requests to all the running applications with the following command:
+The following command forms five connections to the example `nginx` application using the external IP address of the `LoadBalancer` service configuration. The result of the command is a list of those server IP addresses.
+
+* Verify that the load balancer sends requests to all the running applications with the following command:
 +
 [source,terminal]
 ----
 EXTERNAL_IP=192.168.1.241
```

Comment:
Out of scope once again, but if these are things users should enter in the CLI then I think it would need a $ in front:
```suggestion
$ EXTERNAL_IP=192.168.1.241
```

Reply:
This one is deliberate. Commands that are on more than one line can't have the terminal notation on the first line or the copy block only picks up that line.